onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationService.java

```diff
@@ -78,7 +80,7 @@
     private boolean mGetReadyFlag = false;
     private boolean mPullTheCordFlag = false;
 
-    private NavigationServiceProvider mNavProvider;
+    public NavigationServiceProvider mNavProvider;
```

Comment:
Can `mNavProvider` remain `private`?  If so let's change back.

Reply:
We don't need to make the access specifier of mNavProvider public, so changing it back to private.